test/utils/structs/EnumerableMap.behavior.js

```diff
@@ -154,17 +154,6 @@ function shouldBehaveLikeMap(keys, values, zeroValue, methods, events) {
       });
     });
 
```

Comment:
I think we also need to remove these lines:

- [EnumerableMap.test.js:44](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8de6eba8a37d2ece73cb6f0fc7154cdba3619fc0/test/utils/structs/EnumerableMap.test.js#L44)
- [EnumerableMap.test.js:68](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8de6eba8a37d2ece73cb6f0fc7154cdba3619fc0/test/utils/structs/EnumerableMap.test.js#L68)
- [EnumerableMap.test.js:921](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8de6eba8a37d2ece73cb6f0fc7154cdba3619fc0/test/utils/structs/EnumerableMap.test.js#L92)
- [EnumerableMap.test.js:116](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8de6eba8a37d2ece73cb6f0fc7154cdba3619fc0/test/utils/structs/EnumerableMap.test.js#L116)
- [EnumerableMap.test.js:140](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/8de6eba8a37d2ece73cb6f0fc7154cdba3619fc0/test/utils/structs/EnumerableMap.test.js#L140)

Reply:
done